airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -13,6 +13,34 @@ function error() {
 
 PROJECT_DIR=$(pwd)
 
+function configuredbt() {
```

Comment:
Nit: can we leave a comment summarising/explaining what is happening here? It's an intimidating block of code to read and understand due to the long if/else conditions.